scripts/start.e2e.ts

```diff
@@ -1,9 +1,18 @@
 import * as $ from 'shelljs'
 import terminate from 'terminate/promise'
+import yaml from 'js-yaml'
+import fs from 'fs'
 
 describe('start', () => {
   it('should start the preview server successfully', async () => {
-    const child = $.exec('yarn start', { async: true })
+    const envFileContent = yaml.load(
```

Comment:
this change was necessary because I added the first config parameter that was actually requires to be set, and did not default to something in code. this has the nice side effect of adding some coverage for our production config file, since this test will fail if there is a required config that is not included there